build_for_arch.py

```diff
@@ -0,0 +1,12 @@
+import platform
```

Comment:
What's this script is used for ?

Reply:
the script is used to build for multiple archs if and when nuclear decides to support them

Reply:
No need for such abstraction to support something isn't being worked on.

Reply:
and even them you can have modules specific to an architecture. So there would be no need for that.

Reply:
oh okay